galaxy.yml

```diff
@@ -55,9 +55,8 @@ tags:
   - foreman
   - katello
   - satellite
-  - orcharhino
 readme: "README.md"
-homepage: "https://theforeman.org/"
-repository: "https://github.com/theforeman/foreman-ansible-modules"
-documentation: "https://theforeman.org/plugins/foreman-ansible-modules/"
-issues: "https://github.com/theforeman/foreman-ansible-modules/issues"
+homepage: "https://access.redhat.com/products/red-hat-satellite"
+repository: "https://github.com/RedHatSatellite/satellite-ansible-collection"
+documentation: "https://access.redhat.com/documentation/en-us/red_hat_satellite/"
+issues: "https://bugzilla.redhat.com"
```

Comment:
We could make this slightly easier for users with:

https://bugzilla.redhat.com/enter_bug.cgi?product=Red%20Hat%20Satellite%206&component=Ansible%20Collection

Reply:
enter bug or list bugs? :)

Reply:
(https://bugzilla.redhat.com/buglist.cgi?component=Ansible%20Collection&list_id=11042518&product=Red%20Hat%20Satellite%206 so that you don't have to search)

Reply:
Oh, hrmm. I assumed this was "where to go to enter an issue" but now that you bring up listing bugs. I don't know anymore.

Reply:
The list has a "file a bug here" link, so that's probably better?

Reply:
Make it so.